src/Web/View/Base/AbstractView.php

```
@@ -90,7 +90,9 @@ abstract class AbstractView extends Component
     /**
      * The layout file to be used for wrapping rendered views.
      */
-    public private(set) string $layoutFile = '';
+    public private(set) string $layoutFile = '@layout/main' {
```

PHPStan (2.0.3)

Multiple access type modifiers are not allowed on line 93


(phpstan.parse)